jest.setup.mjs

```diff
@@ -1 +1,2 @@
 import '@testing-library/jest-dom';
+import './__mocks__/nextJsRouter.mjs';
```

Comment:
I think you can add the mock files on https://medium.com/@arivu_a/how-to-mock-functions-globally-in-jest-f267fedf7683 (setupFiles) instead of a manual import here

As in general, Jest should import this whenever it believes it should be imported (risk of this being imported when Jest Env is still not prepared)

Reply:
I had not thought about this, thank you.

Perhaps `setupFilesAfterEnv` would be better, then, since those are run after the test framework is installed in the environment, as per the [docs](https://jestjs.io/docs/configuration#setupfilesafterenv-array)?